setup.py

```diff
@@ -11,7 +11,7 @@
 
 
 # Check Python version.
-if sys.version_info < (3, 5):
+if sys.version_info < (3, 7):
```

Comment:
GSW is dropping support for 3.6 and is letting the setup.cfg/pep517/pep518 deal with this: https://github.com/TEOS-10/GSW-Python/pull/55/files#diff-2eeaed663bd0d25b7e608891384b7298L16-L42

You can probably just remove the entire check.